### PR TITLE
Workspace mapping release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -25,6 +25,20 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+:::info
+
+This feature in [Public Preview](feature-previews.md).
+
+:::
+
+<PremiumBadge />
+
+You can now keep teams and projects isolated by authorizing Workspaces to specific clusters. After you enable Workspace authorization for a cluster, only authorized Workspaces can host Deployments in the cluster. Use this feature to gain manage cloud resources by ensuring that only authorized Deployments are running on specific clusters.
```
Comment (chomipi88) [inline astro/release-notes.md:38]:
```suggestion
You can now keep teams and projects isolated by authorizing Workspaces to specific clusters. After you enable Workspace authorization for a cluster, only authorized Workspaces can host Deployments in the cluster. Use this feature to better manage cloud resources by ensuring that only authorized Deployments are running on specific clusters.
```

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 2]
```diff
@@ -25,6 +25,20 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 ## February 14, 2023
 
+### Authorize Workspaces to clusters
+
+:::info
+
+This feature in [Public Preview](feature-previews.md).
```
Comment (kwcanuck) [inline astro/release-notes.md:32]:
```suggestion
This is a [Public Preview](feature-previews.md) feature.
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -25,6 +25,20 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+:::
+
+<PremiumBadge />
+
+You can now keep teams and projects isolated by authorizing Workspaces to specific clusters. After you enable Workspace authorization for a cluster, only authorized Workspaces can host Deployments in the cluster. Use this feature to better manage cloud resources by ensuring that only authorized Deployments are running on specific clusters.
```
Comment (kwcanuck) [inline astro/release-notes.md:38]:
```suggestion
You can now keep teams and projects isolated by authorizing Workspaces to specific clusters. After you enable Workspace authorization for a cluster, only authorized Workspaces can host Deployments in the cluster. Use this feature to better manage cloud resources by ensuring that only authorized Deployments run on specific clusters.
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -25,6 +25,20 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+Configure Workspace authorization in the **Clusters** tab of the Cloud UI.
```
Comment (kwcanuck) [inline astro/release-notes.md]:
```suggestion
To enable Workspace authorization in the  Cloud UI:
1. Click the **Clusters** tab.
2. Click a cluster name in the clusters list.
3. Click the **Workspace Authorization** tab.
4. Click **Edit Workspace Authorization**.
5. Click **Restricted** and then select the Workspaces that can host Deployments in the cluster.
6. Click **Update**.
```
Or, you can link to an existing procedure!

**Review by kwcanuck — APPROVED** [review]

File: astro/release-notes.md [hunk 5]
```diff
@@ -25,6 +27,20 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+This is a [Public Preview](feature-previews.md) feature.
+
+:::
+
+<PremiumBadge />
+
+You can now keep teams and projects isolated by authorizing Workspaces to specific clusters. Use this feature to better manage cloud resources by ensuring that only authorized Deployments are running on specific clusters.
+
+To authorize Workspaces to clusters, see [Authorize Workspaces to a Cluster](modify-cluster.md#authorize-workspaces-to-a-cluster)
```
Comment (jwitz) [inline astro/release-notes.md:42]:
```suggestion
To authorize Workspaces to clusters, see [Authorize Workspaces to a Cluster](modify-cluster.md#authorize-workspaces-to-a-cluster).
```

**Pull request merged**